spec/factories/knights.rb

```diff
@@ -0,0 +1,4 @@
+FactoryGirl.define do
```

Comment:
Yeah, you're right - I'm not using that factory at all in this commit. I used 'rails generate rspec:model knight' to generate this test, so rspec creates that factory automatically. We might need it in the future for other tests, so I left it there for now.